doc/quickstart.rst

```diff
@@ -0,0 +1,488 @@
+
+      deploy serial
+
+**Alternative: Network Flashing via Router**
+
+If you have an OpenWRT router with USB ports and ser2net configured:
+
+.. code-block:: bash
+
+   deploy serial rfc2217://192.168.8.1:5000
+
+(Replace with your router's IP and configured port)
+
+Monitoring Your Node
+====================
+
+After flashing, you can monitor your node in two ways:
+
+Serial Console
+--------------
+
+In a new terminal:
+
+.. code-block:: bash
+
+   iot
+   cd ~/iot-systems/my-home/living-room-lamp
+   console_serial
+
+You'll see debug output and device state changes.
+
+MQTT Messages
+-------------
+
+In another terminal:
+
+.. code-block:: bash
+
+   iot
+   mqtt_listen
+
+Now press the button on your microcontroller. You should see messages like:
+
+.. code-block::
+
+   living-room-lamp/button1 released
+   living-room-lamp/button1 pressed
+
+Congratulations! Your first node is working!
+
+Over-the-Air Updates
+====================
+
+After the initial serial flash, you can update your node over WiFi.
+
+1. Make changes to ``setup.cpp``
+
+2. Simply run:
+
+   .. code-block:: bash
+
+      deploy
+
+No USB cable needed! The node will update itself over WiFi.
+
+Controlling Devices via Node-RED
+================================
+
+Open Node-RED in your browser:
+
+.. code-block::
+
+   http://localhost:40080/nodered
+
+Creating a Simple Flow
+----------------------
+
+1. Drag an **mqtt in** node to the canvas
+2. Double-click it and set the topic to: ``living-room-lamp/button1``
+3. Add a **debug** node and connect the mqtt node to it
+4. Click **Deploy** (top right)
+5. Open the debug panel (bug icon on the right)
+6. Press your physical button - you'll see messages appear!
+
+Controlling the LED
+-------------------
+
+1. Add another **mqtt in** node for ``living-room-lamp/button1``
+2. Add a **change** node:
+   - Set rule 1: Change "pressed" to "on"
+   - Set rule 2: Change "released" to "off"
+3. Add an **mqtt out** node with topic: ``living-room-lamp/blue_led/set``
+4. Connect: mqtt in → change → mqtt out
+5. Click **Deploy**
+
+Now when you press the button, the LED should turn on and off!
+
+Your Second Node
+================
+
+Adding another node is easy - just create another node folder:
+
+.. code-block:: bash
+
+   cd ~/iot-systems/my-home
+   create_node_template bedroom-sensor
+   cd bedroom-sensor
+
+Configure it, flash it, and then you can create flows in Node-RED to make 
+your nodes interact with each other.
+
+For example, a button on one node could control an LED on another node!
+
+For more details, see the `Second Node Tutorial <second-node.rst>`_.
+
+Tips and Best Practices
+=======================
+
+Using iot menu
+--------------
+
+The ``iot menu`` command is your friend for:
+
+- Creating new nodes
+- Deploying nodes
+- Adopting pre-flashed nodes
+- Managing your IoT system
+
+Just run ``iot menu`` in any system or node folder.
+
+Local Editors
+-------------
+
+Use these editors for a better experience:
```

Comment:
suggest also to use a full fledged IDE like vscode or vscodium (also zed works well). Especailly in VSCode you get some superpowers for debugging advanced problems as in the buold folder you will find a full platformio project.